riscv_isac/coverage.py

```diff
@@ -955,7 +973,7 @@ def compute(trace_file, test_name, cgf, parser_name, decoder_name, detailed, xle
     #This flen value is being used in compute_per_line method to build the the  string, in-order to cross-veriy the coverpoints
     arch_state = archState(xlen,32)
```

Comment:
Hardcoded flen values show that there is no support for D extension. The `flen` should be derived similar to `xlen` i.e from the command line.

Reply:
Again, this is not in the scope of this PR. But ok to make this change.